diff --git a/node_modules/@1stg/eslint-config/base.js b/node_modules/@1stg/eslint-config/base.js
index 5eef2be..b342f5d 100644
--- a/node_modules/@1stg/eslint-config/base.js
+++ b/node_modules/@1stg/eslint-config/base.js
@@ -32,7 +32,7 @@ export const base = tseslint.config([
   promise.configs['flat/recommended'],
   regexp.configs['flat/recommended'],
   sonarjs.configs.recommended,
-  unicorn.configs.recommended,
+  unicorn.configs['flat/recommended'],
   prettierExtends,
   {
     name: '@1stg/node',
diff --git a/node_modules/@1stg/eslint-config/overrides.js b/node_modules/@1stg/eslint-config/overrides.js
index 9cdf80b..c9247d6 100644
--- a/node_modules/@1stg/eslint-config/overrides.js
+++ b/node_modules/@1stg/eslint-config/overrides.js
@@ -15,7 +15,7 @@ import vitest_ from '@vitest/eslint-plugin'
 import angular_ from 'angular-eslint'
 import angularTemplateParser from 'angular-eslint-template-parser'
 import { createTypeScriptImportResolver } from 'eslint-import-resolver-typescript'
-import eslintMdx from 'eslint-mdx'
+import * as eslintMdx from 'eslint-mdx'
 import importX from 'eslint-plugin-import-x'
 import jest_ from 'eslint-plugin-jest'
 import jsdoc from 'eslint-plugin-jsdoc'
