diff --git a/node_modules/@1stg/eslint-config/base.js b/node_modules/@1stg/eslint-config/base.js
index f581732..8c5f519 100644
--- a/node_modules/@1stg/eslint-config/base.js
+++ b/node_modules/@1stg/eslint-config/base.js
@@ -134,7 +134,7 @@ export const base = tseslint.config([
         },
       ],
       'no-negated-condition': 2,
-      'no-process-exit': 0, // suspended by unicorn-x/no-process-exit
+      'no-process-exit': 0, // suspended by unicorn/no-process-exit
 
       // The following rules are duplicate with `eslint-plugin-import-x`
       'n/no-extraneous-import': 0,
@@ -149,15 +149,15 @@ export const base = tseslint.config([
       'prefer-const': [2, { destructuring: 'all' }],
       'prefer-object-spread': 2,
       'simple-import-sort/exports': 2,
-      'unicorn-x/catch-error-name': [
+      'unicorn/catch-error-name': [
         2,
         {
           name: 'error',
           ignore: ['^e(rr)?$'],
         },
       ],
-      'unicorn-x/consistent-function-scoping': 0,
-      'unicorn-x/filename-case': [
+      'unicorn/consistent-function-scoping': 0,
+      'unicorn/filename-case': [
         2,
         {
           cases: {
@@ -168,13 +168,13 @@ export const base = tseslint.config([
           ignore: [/^[A-Z](([\dA-Z]+_)*[\dA-Z]+)?\.(mdx?|ya?ml)$/],
         },
       ],
-      'unicorn-x/no-array-reduce': 0,
-      'unicorn-x/no-null': 0,
-      'unicorn-x/no-unreadable-array-destructuring': 0, // conflict with `no-unused-vars`
-      'unicorn-x/prefer-module': 0,
-      'unicorn-x/prefer-object-from-entries': 0,
-      'unicorn-x/prevent-abbreviations': 0,
-      'unicorn-x/prefer-export-from': [2, { ignoreUsedVariables: true }],
+      'unicorn/no-array-reduce': 0,
+      'unicorn/no-null': 0,
+      'unicorn/no-unreadable-array-destructuring': 0, // conflict with `no-unused-vars`
+      'unicorn/prefer-module': 0,
+      'unicorn/prefer-object-from-entries': 0,
+      'unicorn/prevent-abbreviations': 0,
+      'unicorn/prefer-export-from': [2, { ignoreUsedVariables: true }],
     },
   },
   {
