diff --git a/node_modules/@1stg/eslint-config/overrides.js b/node_modules/@1stg/eslint-config/overrides.js
index 7353a01..979b261 100644
--- a/node_modules/@1stg/eslint-config/overrides.js
+++ b/node_modules/@1stg/eslint-config/overrides.js
@@ -223,9 +223,15 @@ exports.ts = [
       'n/shebang': 0,
     },
   },
+  {
+    files: '**/*.{md,mdx}/**/*.ts',
+    parserOptions: {
+      project: null,
+    },
+  },
   {
     files: ['*.cts', '*.mts', '*.ts', '*.tsx'],
-    excludedFiles: ['*.d.cts', '*.d.mts', '*.d.ts'],
+    excludedFiles: ['**/*.{md,mdx}/**/*.ts', '*.d.cts', '*.d.mts', '*.d.ts'],
     extends: ['plugin:@typescript-eslint/recommended-requiring-type-checking'],
     rules: {
       '@typescript-eslint/no-floating-promises': [
