diff --git a/node_modules/@1stg/eslint-config/overrides.js b/node_modules/@1stg/eslint-config/overrides.js
index 2697512..f6db0b7 100644
--- a/node_modules/@1stg/eslint-config/overrides.js
+++ b/node_modules/@1stg/eslint-config/overrides.js
@@ -503,7 +503,7 @@ exports.json = {
   ],
   extends: [
     // eslint-disable-next-line sonarjs/no-duplicate-string
-    'plugin:jsonc/auto-config',
+    // 'plugin:jsonc/auto-config',
     'plugin:jsonc/recommended-with-json',
     // eslint-disable-next-line sonarjs/no-duplicate-string
     'plugin:json-schema-validator/recommended',
@@ -522,7 +522,7 @@ exports.jsonc = {
     'tsconfig.*.json',
   ],
   extends: [
-    'plugin:jsonc/auto-config',
+    // 'plugin:jsonc/auto-config',
     'plugin:jsonc/recommended-with-jsonc',
     'plugin:json-schema-validator/recommended',
     'plugin:jsonc/prettier',
@@ -532,7 +532,7 @@ exports.jsonc = {
 exports.json5 = {
   files: '*.json5',
   extends: [
-    'plugin:jsonc/auto-config',
+    // 'plugin:jsonc/auto-config',
     'plugin:jsonc/recommended-with-json5',
     'plugin:json-schema-validator/recommended',
     'plugin:jsonc/prettier',
