diff --git a/node_modules/unified-engine/index.d.ts b/node_modules/unified-engine/index.d.ts
index 1f8706e..9f1cc30 100644
--- a/node_modules/unified-engine/index.d.ts
+++ b/node_modules/unified-engine/index.d.ts
@@ -1,4 +1,5 @@
 export { Configuration } from "./lib/configuration.js";
+export { Ignore } from "./lib/Ignore.js";
 export { engine } from "./lib/index.js";
 export type ConfigResult = import("./lib/configuration.js").ConfigResult;
 export type Completer = import("./lib/file-set.js").Completer;
diff --git a/node_modules/unified-engine/index.js b/node_modules/unified-engine/index.js
index 4c993a8..d96f4ce 100644
--- a/node_modules/unified-engine/index.js
+++ b/node_modules/unified-engine/index.js
@@ -13,5 +13,6 @@
 
 // it’s mostly private, but useful for tools like `eslint-mdx`.
 export {Configuration} from './lib/configuration.js'
+export {Ignore} from './lib/ignore.js'
 
 export {engine} from './lib/index.js'
diff --git a/node_modules/unified-engine/lib/ignore.d.ts b/node_modules/unified-engine/lib/ignore.d.ts
index b2329db..7d35ff1 100644
--- a/node_modules/unified-engine/lib/ignore.d.ts
+++ b/node_modules/unified-engine/lib/ignore.d.ts
@@ -37,19 +37,19 @@ export type Options = {
     /**
      *   Whether to detect ignore files.
      */
-    detectIgnore: boolean | undefined;
+    detectIgnore?: boolean | undefined;
     /**
      *   Basename of ignore files.
      */
-    ignoreName: string | undefined;
+    ignoreName?: string | undefined;
     /**
      *   Explicit path to an ignore file.
      */
-    ignorePath: URL | string | undefined;
+    ignorePath?: URL | string | undefined;
     /**
      *   How to resolve.
      */
-    ignorePathResolveFrom: ResolveFrom | undefined;
+    ignorePathResolveFrom?: ResolveFrom | undefined;
 };
 /**
  * How to resolve.
diff --git a/node_modules/unified-engine/lib/ignore.js b/node_modules/unified-engine/lib/ignore.js
index be6d078..da877af 100644
--- a/node_modules/unified-engine/lib/ignore.js
+++ b/node_modules/unified-engine/lib/ignore.js
@@ -16,13 +16,13 @@
  *   Configuration.
  * @property {string} cwd
  *   Base.
- * @property {boolean | undefined} detectIgnore
+ * @property {boolean | undefined} [detectIgnore]
  *   Whether to detect ignore files.
- * @property {string | undefined} ignoreName
+ * @property {string | undefined} [ignoreName]
  *   Basename of ignore files.
- * @property {URL | string | undefined} ignorePath
+ * @property {URL | string | undefined} [ignorePath]
  *   Explicit path to an ignore file.
- * @property {ResolveFrom | undefined} ignorePathResolveFrom
+ * @property {ResolveFrom | undefined} [ignorePathResolveFrom]
  *   How to resolve.
  *
  * @typedef {'cwd' | 'dir'} ResolveFrom
