diff --git a/node_modules/wrap-ansi-cjs/index.js b/node_modules/wrap-ansi-cjs/index.js
index d502255..c33a23a 100755
--- a/node_modules/wrap-ansi-cjs/index.js
+++ b/node_modules/wrap-ansi-cjs/index.js
@@ -1,6 +1,6 @@
 'use strict';
-const stringWidth = require('string-width');
-const stripAnsi = require('strip-ansi');
+const stringWidth = require('string-width-cjs');
+const stripAnsi = require('strip-ansi-cjs');
 const ansiStyles = require('ansi-styles');
 
 const ESCAPES = new Set([
